examples/with-magic/pages/login.js

```diff
@@ -21,7 +21,7 @@ const Login = () => {
     }
 
     try {
-      const magic = new Magic(process.env.NEXT_PUBLIC_MAGIC_PUBLISHABLE_KEY)
+      const magic = new Magic(process.env.NEXT_EXAMPLE_MAGIC_PUBLISHABLE_KEY)
```

Comment:
I believe this still needs to be prefixed with `NEXT_PUBLIC_` to be exposed to the client-side.

Reply:
Oh right! Thanks for catching that 👍